README.md

```diff
@@ -9,18 +9,24 @@ This repository is licensed under the [Apache 2.0 License](https://github.com/ma
 ## Table of Contents
 
 - [Admin Guide](#admin-guide)
+  - [Prerequisites](#prerequisites)
   - [Setting up](#setting-up)
   - [Notification management](#notification-management)
   - [System Admin slash commands](#system-admin-slash-commands)
 - [User's Guide](#users-guide)
   - [Slash commands](#slash-commands)
   - [Create a ticket](#create-a-ticket)
 - [Development](#development)
+- [Provision](#provision)
 
 ## Admin Guide
 
 This guide is intended for Mattermost System Admins setting up the Zendesk app. For more information about contributing to this plugin, visit the [Development section](#development).
 
+### Prerequisites
+
+The Zendesk App is designed using the [Apps Framework](https://developers.mattermost.com/integrate/apps) which differs from the Plugin Framework. The Apps framework is written as a [Mattermost plugin](https://github.com/mattermost/mattermost-plugin-apps) and needs to be installed and enabled before Mattermost Apps like the Zendesk App can be installed. You can install the Apps plugin via the Mattermost Marketplace.
```

Comment:
To be more accurate the changes in the webapp, mobile and server repos (to not mention the provisioning work for the cloud - which are part of the hosting aspect of the framework) are still part of (supporting) the Apps Framework , so maybe we should not equate Apps Framework with Apps Plugin.